### ISSUE TITLE
Speaking commands in ATC (suggestion) (proof of concept)

### ISSUE THREAD

**Issue by sixeyeco** [zlsa/atc#360]
If this were perfected and implemented I feel it would make an already awesome game even better. 

I used speechpad.pw and it appears that the developer is very responsive.

 [See "proof of concept" video](https://youtu.be/hQVKTt2W9yM)

**Comment by Maverick283** [comment]
There was something similar already done in #99 yet it never got merged (Don't know why not).

**Comment by sixeyeco** [comment]
I searched before posting, sorry for the dupe. [#99](https://github.com/zlsa/atc/pull/99) doesn't appear to have much in the way of supporters. I guess I'm just tired of my own darn typos more than anything. Maybe there's still hope for [scrolling recent commands](https://github.com/zlsa/atc/issues/341).

**Comment by Maverick283** [comment]
oh dont close it!!! :(

my intention was only to point put that there is a base so the programming might be easier if it can be build on! i very much like the idea, together with #365 this could be awesome!

**Comment by erikquinn** [comment]
Yeah, #99 has been around for a while; maybe the no-merge is because it never got thoroughly tested, and now it may have numerous conflicts (as it is a large change, adding 1,101 lines). If those merge conflicts could get resolved, though, and some thorough testing done on it, I would think it has a good chance of being added as an experimental function.

**Comment by sixeyeco** [comment]
@Maverick283 Oops. Sorry. I was being overly conscientious.

Issues that I've noticed is that it misinterprets "wait" as 'weight" and cannot seem distinguish call signs  as a single "word". With careful diction however, I achieved an encouraging degree of success.

Perhaps it's possible to severely limit the vocabulary that the recognition software can even try to transcribe. For example, if the word "weight" were omitted from "the list" then it would correctly type "wait". This would also address issues that would arise from voice typing fixes, whose spelling can be somewhat...unconventional.
